content/blogs/2024-01-08-sentry-plugin.md

```diff
@@ -0,0 +1,550 @@
+- `SUCCESS`: The Execution or Task Run has been completed successfully.
+- `WARNING`: The Execution or Task Run exhibited unintended behavior, but the execution continued and was flagged with a warning.
+- `FAILED`: The Execution or Task Run exhibited unintended behavior that caused the execution to fail.
+- `KILLING`: A command was issued that asked for the Execution or Task Run to be killed. The system is in the process of killing the associated tasks.
+- `KILLED`: An Execution or Task Run was killed (upon request), and no more tasks will run for it.
+- `RESTARTED`: This status is transitive. It is the same as CREATED, but for a flow that has already been executed, failed, and has been restarted.
+
+## Putting it all together
+
+Finally, let’s look at a production quality example using **SentryAlert** and **SentryExecution**.
+
+### SentryAlert Production Example
+The flow below is a production quality example using the SentryAlert task to send an issue when the get_files task throws an error. This example uses secrets, variable and taskDefaults for the db connection values and the Sentry DSN.
+
+```yaml
+id: gen_postgresql_error_sentry_alert
+namespace: sentry_article
+description: Generate an error so we can send a SentryAlert
+errors:
+  - id: log_msg_on_err
+    type: io.kestra.core.tasks.log.Log    
+    message: "Something bad happened while running {{execution.id}} {{task.id}}"
+  - id: send_sentry_alert_on_failure
+    type: io.kestra.plugin.notifications.sentry.SentryAlert
+    dsn: "{{ secret('SENTRY_DSN') }}"
+    payload: |
+      {
+        "culprit": "Deliberately bad SQL.",
+        "message": "Kestra POSTGRES: FAIL DEMO",
+        "timestamp": "{{ execution.startDate }}",
+        "platform": "java",
+        "level": "error",
+        "transaction": "/execution/id/{{ execution.id }}",
+        "server_name": "localhost:8080",
+        "extra": {
+          "Title": "This is the Title",
+          "Namespace": "{{ flow.namespace }}",
+          "Flow ID": "{{ flow.id }}",
+          "Execution ID": "{{ execution.id }}",
+          "Link": "http://localhost:8080/ui/executions/{{flow.namespace}}/{{flow.id}}/{{execution.id}}"
+        }      
+      }
+
+tasks:
+- id: get_files
+  type: io.kestra.plugin.jdbc.postgresql.Query
+  sql: select not_a_real_column from not_a_real_table;
+  fetch: true
+
+- id: use_files
+  type: io.kestra.core.tasks.log.Log
+  message: Number of rows returned is "{{get_files.outputs.size}}"
+
+triggers:
+  - id: schedule
+    type: io.kestra.core.models.triggers.types.Schedule
+    cron: "*/05 * * * *"
+    # every 5 minutes
+
+variables:
+    db_host: "host.docker.internal"
+    db_name: "summ_dev"
+
+taskDefaults:
+  - type: io.kestra.plugin.jdbc.postgresql.Query
+    values:
+      username: "{{ secret('PG_USERNAM') }}"
+      password: "{{ secret('PG_PWD') }}"
+      url: jdbc:postgresql://{{vars.db_host}}:5432/{{ vars.db_name}}
+     
+disabled: false
```

Comment:
```suggestion
```

delete disabled

also the order is a bit not typical, maybe try the typical order of properties?

- id
- namespace
- description
- variables
- tasks
- triggers
- taskDefaults
- errors